src/doc/dev_manual/test_suite.rst

```diff
@@ -3,20 +3,20 @@ Regression Testing
 
 Overview
 --------
-VisIt_ has a large and continually growing test suite. VisIt_'s test
-suite involves a combination python scripts
-in ``src/test``, raw data and data generation sources in ``src/testdata``
-and of course the VisIt_ sources themselves. Regression tests are
-run on a nightly basis. Testing exercises VisIt_'s viewer,
-mdserver, engine and cli but not the GUI.
+VisIt_ has a large and continually growing test suite.
+VisIt_'s test suite involves a combination of python scripts in ``src/test``, raw data and data generation sources in ``src/testdata`` and of course the VisIt_ sources themselves.
+Regression tests are run on a nightly basis.
+Testing exercises VisIt_'s viewer, mdserver, engine and cli but not the GUI.
 
 
 Running regression tests
 ------------------------
 
 Where nightly regression tests are run
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-The regression suite is run on `LLNL's Pascal Cluster <https://hpc.llnl.gov/hardware/platforms/pascal>`_. Pascal runs the TOSS3 operating system, which is a flavor of Linux. If you are going to run the regression suite yourself you should run on a similar system or there will be differences due to numeric precision issues.
+The regression suite is run on `LLNL's Pascal Cluster <https://hpc.llnl.gov/hardware/platforms/pascal>`_.
+Pascal runs the TOSS3 operating system, which is a flavor of Linux.
+If you are going to run the regression suite yourself you should run on a similar system or there will be differences due to numeric precision issues.
```

Comment:
Should we mention the *fuzz* knobs we provide and a link to the section that describes them?